Promtail pull cloudflare logs (#4813)
grafana/loki: docs/sources/clients/promtail/scraping.md

        target_label: facility
```

## Cloudflare

> Promtail supports pulling http log messages from Cloudflare using the [Logpull API](https://developers.cloudflare.com/logs/logpull).
> The Cloudflare targets can be configured using the `cloudflare` stanza:
> 
> ```yaml
> scrape_configs:
> - job_name: cloudflare
>   cloudflare:
>     api_token: REDACTED
>     zone_id: REDACTED
>     fields_type: all
>     labels:
>       job: cloudflare-foo.com
> ```
> 
> Only the `api_token` and `zone_id` is required.
> see the [configuration](../../configuration/#cloudflare) section for more information.

Reviewer: ```suggestion
Promtail supports pulling HTTP log messages from Cloudflare using the [Logpull API](https://developers.cloudflare.com/logs/logpull).
The Cloudflare targets can be configured with a `cloudflare` block:

```yaml
scrape_configs:
- job_name: cloudflare
  cloudflare:
    api_token: REDACTED
    zone_id: REDACTED
    fields_type: all
    labels:
      job: cloudflare-foo.com
```

Only `api_token` and `zone_id` are required.
Refer to the [Cloudfare](../../configuration/#cloudflare) configuration section for details.
```
